### DISCUSSION TITLE
Github Desktop - push to current branch?

### DISCUSSION THREAD

**Discussion by tonymeadors** [community/community#23725] [New to GitHub]
<p>I have 2 branches master and Develop.<br>
I commit several changes to Develop<br>
and then of course want to push them to the Remote Develop.<br>
But the button is a Push to Origin -  (sending them to master there)</p>
<p>Silly question - how do I push to the proper branch<br>
using Github Desktop?</p>
<p>thanks,<br>
tonyM</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/tonymeadors">@tonymeadors</a>,</p>
<p>What branch does GitHub Desktop say that you’re currently on? If you’re on the Develop branch, it should push to the Develop branch on the remote server instead of sending them to the Master branch.</p>

**Comment by tonymeadors** [comment]
<p>Thanks.<br>
I tried it again that way and although the button still says<br>
Push to Origin it went to Develop as hoped ready for merge.<br>
thanks,<br>
tonyM</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>“Origin” refers to the remote repository as a whole, not just the particular branch you’re on. That’s why it still says “Push to Origin.”</p>
